api/fixtures/example.go

```diff
@@ -340,6 +356,46 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 					IOPS: o.AWS.RootVolumeIOPS,
 				},
 			}
+		case hyperv1.KubevirtPlatform:
+			runAlways := kubevirtv1.RunStrategyAlways
+			guestQuantity := apiresource.MustParse(o.Kubevirt.Memory)
+			nodePool.Spec.Platform.Kubevirt = &hyperv1.KubevirtNodePoolPlatform{
+				NodeTemplate: &capikubevirt.VirtualMachineTemplateSpec{
+					Spec: kubevirtv1.VirtualMachineSpec{
+						RunStrategy: &runAlways,
```

Comment:
In general I would even consider changing it in kubevirt itself, so that we restart the VMI in-place. Rescheduling has a lot of overhead. All other k8s controllers avoid that and restart in-place. Potentially has impact on how professional your monitoring has to be though.